boundary_layer/builders/util.py

```diff
@@ -120,7 +120,7 @@ def format_value(value):
 
         return '{{ {items} }}'.format(items=','.join(pairs))
 
-    if isinstance(value, (int, float, type(None))):
+    if isinstance(value, (int, float, type(None), bytes)):
```

Comment:
If we want to support bytes, we should just return the bytes and not pass it through the str function. e.g.,

if isinstance(value, bytes):
    return bytes

Reply:
@vchiapaikeo I considered that, however since this function is returning exclusively strs apart from this, I wanted to keep the return type uniform. I understand that now with jinja this is not causing problems. However, what if someone decorated this method call with more logic thinking they were getting a str and instead got bytes? Maybe this is not a problem and i am overthinking